### DISCUSSION TITLE
Maintence Mode

### DISCUSSION THREAD

**Discussion by milangamesonyt** [Paymenter/Paymenter#290] [Ideas]
Maybe add a button in the settings tab to enable/disable a maintenance mode and maybe just like WHMCS make it that when  you try to visit the site and your not logged in to a admin account that you cant visit the site

**Comment by Moreau741** [comment]
Fantastic suggestion! Additionally, you can employ IP blocking, which restricts access to the panel solely to your own IP address through the firewall.
